infra/Pos/Infra/Configuration.hs

```diff
@@ -67,6 +67,8 @@ infraConfiguration = given
 
 ntpServers :: [String]
 ntpServers =
-    [ "time.windows.com"
-    , "clock.isc.org"
-    , "ntp5.stratum2.ru" ]
+    [ "0.pool.ntp.org"
+    , "2.pool.ntp.org"
+    , "3.pool.ntp.org"
+    ]
+    -- need only 3 servers :shrug:
```

Comment:
http://www.pool.ntp.org/en/use.html

For future reference, we should be able to use all 4, even if on any single try we only use a subset.